### DISCUSSION TITLE
Sign up problem

### DISCUSSION THREAD

**Discussion by Sankalp-sa** [camicroscope/GSOC#14] [Q&A] [answered]
Hi @birm as you told me earlier to write ["**"] in list of filters to sign in so after sign in i am getting this in console. So, what to do next ?
![image](https://user-images.githubusercontent.com/102241865/224505814-1fd54b72-991c-4b68-a75d-60bdf55ae0a6.png)
![image](https://user-images.githubusercontent.com/102241865/224505862-6d02aff3-93f3-4de8-aa40-b571406fdb33.png)

**Comment by Sankalp-sa** [comment]
Also, I saw that its a bug in https://github.com/camicroscope/caMicroscope/issues/472 

So, do we have to remove this input field or we have to find an alternative of it ?

**Comment by Sankalp-sa** [comment, marked as answer]
I re-built the environment and the issue is solved. Sorry for the trouble.

**Comment by birm** [comment]
Glad you got it resolved! The relevant code as to why I suggested "**" is here, in case you're curious: https://github.com/camicroscope/Caracal/blob/945e448e8e03ac72c26a7b4eee2b6f2b41f7d725/handlers/authHandlers.js#L214
